web/crux-ui/src/components/products/versions/deployments/deployment-container-status-list.tsx

```diff
@@ -50,51 +53,35 @@ const DeploymentContainerStatusList = (props: DeploymentContainerStatusListProps
       return weak
     }
 
-    const prefix = `${deployment.prefix}-`
-    const containerNames = strong.map(it => {
-      if (it.id && it.name.startsWith(prefix)) {
-        // came from docker
-        return it.name.substring(prefix.length, it.name.length)
-      }
+    const containerNames = strong.map(it => containerNameOf(it.id))
```

Comment:
I think this is an alread existing bug but I noticed it here:
Containers in the message has no prefix, instead the whole container name comes as `name`, so `containerNameOf` just appends a `-` in front which results in the status list showing the container as not found.